doc/latest_news.html

```diff
@@ -4,6 +4,7 @@
       December 22nd 2015: HoloViews 1.4.1 released and now available on 
       <a href="https://pypi.python.org/pypi/holoviews/">PyPI</a> and
       <a href="https://anaconda.org/anaconda/holoviews">Anaconda</a>.
+      Now includes full support for the <a href="http://bokeh.pydata.org">Bokeh</a> plotting library.
```

Comment:
I don't think the plotly backend is going to be merged any time soon. I would just drop the word 'full' to keep it simple (I agree 'official' commits us to too much) so just 'Now includes support ...'